### PR TITLE
Linguist Trait & 2 Languages

### PR DESCRIPTION
**Pull request by SouthyOsmanli**
## About The Pull Request
<details>
<summary>
	Adds an upbringing trait called "Aspiring Linguist", allowing a selection of another language upon spawn. As requested by a few players for a Latin based language, similar to Spanish/Italian, added in Esperanto - and another request for a Arabic of sorts was the reason for Yassari. Said I'd add these if I figured out a linguist trait that allowed for an extra-secondary language as a trade-off between an actually useful perk/stats (like rough-life, idiot savant, etc) - so here you go.
</summary>
<hr>
	
<hr>
</details>

## Changelog
:cl:
add: Linguist background trait.
add: Yassari & Esperanto languages.
/:cl:

### CODE REVIEW

File: code/datums/perks/job.dm [hunk 1]
```diff
@@ -193,6 +193,38 @@
...
+	options["Latin"] = LANGUAGE_LATIN
+	var/choice = input(M,"What language do you know?","Linguist Choice") as null|anything in options
+	if(src && choice)
+		M.add_language(choice)
+		M.stats.removePerk(/datum/perk/linguist)
+	return TRUE
+
+	if(anti_cheat)
+		return FALSE
+	anti_cheat = TRUE
```
Comment (r4d6) [inline code/datums/perks/job.dm:223]: This part of the code will never activate, because it return TRUE before.

**Comment by Kazkin** [comment]
Gonna hold off on this one, pretty sure we got enough languages but I'd like opinions on it.

File: code/datums/perks/job.dm (hunk 1 above)
Comment (SouthyOsmanli) [reply code/datums/perks/job.dm:223]: Adjusted.

**Comment by Trilbyspaceclone** [comment]
I like this pr and support it

**Comment by ThePainkiller** [comment]
Adding Esperanto as a "spanish/italian conlang" makes as much sense as adding portuguese and calling it "japanese/finnish creole".  It's literally more akin of mixing french, german and polish into one language, and sounds nothing like either italian/latin, let alone spanish (which I am a natural speaker of). The intention of adding romance languages is appreciated, but it does not (nor will it) sound like a romance language at all.

**Comment by cdb-is-not-good** [comment]
I am strongly in favor. Tradeoff for more language=compelling choice, good stuff. 
new languages? Neat! The ability to select your language later in the round allows opportunities for interpretation(and can tamp down on the 'cliqueishness' that can form around a less used language). 
Plus, we did need for an Islamic language for all those muslim Blackshield.
Inshallah this PR is merged.

**Comment by Kazkin** [comment]
Given the support, merged.

**Pull request merged**